package.json

```diff
@@ -43,6 +43,7 @@
   "dependencies": {
     "@emotion/core": "^10.0.15",
     "@emotion/styled": "^10.0.15",
+    "@types/react": "16.8.23",
```

Comment:
I did it because using types as a utility and interface seems to be in line with you need this for development. Unless I have dependencies/devDependencies wrong.